src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishTrimmedWindowsFormsAndWPFApps.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.NET.Build.Tasks;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Publish.Tests
+{
+    public class GivenThatWeWantToPublishTrimmedWindowsFormsAndWPFApps : SdkTest
+    {
+        public GivenThatWeWantToPublishTrimmedWindowsFormsAndWPFApps(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [WindowsOnlyRequiresMSBuildVersionFact("16.8.0")]
+        public void It_publishes_and_err_windows_forms_app()
+        {
+            var targetFramework = "net6.0-windows";
+            TestProject testProject = new TestProject()
+            {
+                Name = "WinformsFailTest",
+                ProjectSdk = "Microsoft.NET.Sdk", 
```

Comment:
nit: we don't need to specify the sdk when it's just the default, this line can be removed in all the tests.